test/services/batchaiManagement/batchaiManagementClient-tests.js

```diff
@@ -0,0 +1,896 @@
+          }
+        },
+        nodeSetup: {
+          // It's possible to mount multiple Azure File shares and Azure Storage containers on each compute node.
+          mountVolumes: {
+            azureFileShares: [
+              {
+                azureFileUrl: util.format('https://%s.file.core.windows.net/%s', storageAccountName, azFileShareName),
+                directoryMode: '0777',
+                fileMode: '0777',
+                relativeMountPath: azFileShareMountName,  // Content of the file share be available at $AZ_BATCHAI_MOUNT_ROOT/azfs
+                accountName: storageAccountName,
+                credentials: {
+                  accountKey: storageAccountKey
+                }
+              }
+            ],
+            azureBlobFileSystems: [
+              {
+                containerName: azContainerName,
+                relativeMountPath: azContainerMountName, // Content of the container be available at $AZ_BATCHAI_MOUNT_ROOT/azcs
+                accountName: storageAccountName,
+                credentials: {
+                  accountKey: storageAccountKey
+                }
+              }
+            ]
+          }
+        },
+        userAccountSettings: {
+          adminUserName: 'username',
+          adminUserPassword: uuid.v4(),
+          adminUserSshPublicKey: sshPublicKey
+        }
+      };
+
+      client.clusters.create(groupName, 'cluster3', cluster, function (err, result) {
+        should.not.exist(err);
+        should.equal(result.name, 'cluster3');
+        should.equal(result.provisioningState, 'succeeded');
+        should.equal(result.vmSize, 'STANDARD_D1');
+        should.equal(result.currentNodeCount, 0);  // Shows the number of nodes currently allocated.
+        should.equal(result.scaleSettings.manual.targetNodeCount, 0);  // Number of nodes which will be allocated.
+        should.equal(result.userAccountSettings.adminUserName, 'username');
+        should.not.exist(result.userAccountSettings.adminUserPassword);  // Batch AI will never report the password.
+        should.equal(result.nodeSetup.mountVolumes.azureBlobFileSystems.length, 1);
+        should.equal(result.nodeSetup.mountVolumes.azureBlobFileSystems[0].accountName, storageAccountName);
+        should.equal(result.nodeSetup.mountVolumes.azureBlobFileSystems[0].containerName, azContainerName);
+        should.equal(result.nodeSetup.mountVolumes.azureBlobFileSystems[0].relativeMountPath, azContainerMountName);
+        should.not.exist(result.nodeSetup.mountVolumes.azureBlobFileSystems[0].credentials.accountKey);  // Never reported.
+        should.equal(result.nodeSetup.mountVolumes.azureFileShares.length, 1);
+        should.equal(result.nodeSetup.mountVolumes.azureFileShares[0].accountName, storageAccountName);
+        should.equal(result.nodeSetup.mountVolumes.azureFileShares[0].azureFileUrl,
+          util.format('https://%s.file.core.windows.net/%s', storageAccountName, azFileShareName));
```

Comment:
indentation

Reply:
done

Reply:
actually, it's continuation of the previous line, so identation is correct?

Reply:
ok